pom.xml

```diff
@@ -154,6 +154,8 @@
         <module>authentication</module>
         <module>auth-table-based-service</module>
         <module>health-checker</module>
+        <module>config-store-api</module>
+        <module>config-store-k8s-config-map</module>
```

Comment:
This can just be `config-store-file-based` or something like that. In k8s a configmap is just a way to attach a file to a pod. Once the file is attached then your application uses the file as if it were any other local file.